fastlane/metadata/android/en-US/changelogs/100000.txt

```diff
@@ -0,0 +1,31 @@
+First official release under a new name after forking [Hayai Launcher](https://github.com/edgarvperes/HayaiLauncher).
```

Comment:
Markdown doesn't seem to be supported, however for simplicity I decided to leave markdown here as is

If html tags are supported (like in full_description.txt), then we could render markdown to html.

@IzzySoft , do you know if html tags in changelog entries are allowed? Thanks!